regression-tests/sparktkregtests/testcases/frames/frame_timeseries_test.py

```diff
@@ -33,6 +33,8 @@
 import statsmodels.stats.diagnostic as smd
 import statsmodels.formula.api as smf
 import statsmodels.regression.linear_model as smrl
+import sklearn.metrics
+from sklearn import linear_model
 
 #Change filename to the location of the csv file used for the test
 filename = "../datasets/timeseriesstats.csv"
```

Comment:
There is a helper function to calculated this dynamically. I don't believe this will work in automation